package-lock.json

```diff
@@ -12056,7 +12056,7 @@
         "@types/chai-as-promised": "^7.1.8",
         "@types/sinon": "^17.0.3",
         "@types/sinon-chai": "^3.2.12",
-        "@types/sparql-http-client": "^3.0.0",
+        "@types/sparql-http-client": "^3.0.4",
```

Comment:
This is the main change, actually, having the types allow creating one instance from another directly